cmd/aries-js-worker/src/aries.js

```diff
@@ -176,90 +176,255 @@ const Aries = function(opts) {
             return () => {quit = true}
         },
 
+        /**
+         * DIDExchange methods - Refer to [OpenAPI spec](docs/rest/openapi_spec.md#generate-openapi-spec) for
+         * input params and output return json values.
+         */
         didexchange: {
             pkgname: "didexchange",
-            createInvitation: async function (text) {
+
+            /**
+             * Creates a DID Exchange Invitation.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
```

Comment:
I guess there isn't a good way to indicate that the Promise is for the JSON response from the Controller?

Reply:
Should instead be `Promise<Object>~? Unless we start defining specific types for the models.... (nah)

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/415b0b2413e6b4948632dec5a47ce53406011239..eeb7de2aa3ffaab86cf1542c90f7c65569c8fd2c#diff-a43de5ddb6dcf66800367a9232d43f90R123) to `Promise<Object>`